### PR TITLE
HDDS-3850. Update the admin document to let user know how to show the status of all rules.

### PR DESCRIPTION
**Pull request by maobaolong**
## What changes were proposed in this pull request?

Update admin document, show the `--verbose` option can show the status of all rules.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3850

## How was this patch tested?

No need.

### CODE REVIEW

**Comment by maobaolong** [comment]
@dineshchitlangia Please take a look at this document updated, with this update, user can know the `--verbose` option.

File: hadoop-hdds/docs/content/tools/Admin.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Ozone Admin command (`ozone admin`) is a collection of tools intended to be used
 
 And quick overview about the available functionalities:
 
- * `ozone admin safemode`: You can check the safe mode status and force to leave/enter from/to safemode.
+ * `ozone admin safemode`: You can force to leave/enter from/to safemode and check the safe mode status, with `--verbose` option, will print status of all rules.
```
Comment (dineshchitlangia) [inline hadoop-hdds/docs/content/tools/Admin.md:27]:
```suggestion
 * `ozone admin safemode`: You can check the safe mode status and force to leave/enter from/to safemode,  `--verbose` option will print validation status of all rules that evaluate safemode status.
```

**Comment by maobaolong** [comment]
@dineshchitlangia Thank you for your suggestion. PTAL.

**Comment by dineshchitlangia** [comment]
Thanks @maobaolong for the contribution.

**Pull request merged**